coral/src/app/layout/header/HeaderMenuLink.tsx

```diff
@@ -1,33 +1,23 @@
 import { Icon, Tooltip } from "@aivenio/aquarium";
```

Comment:
Note: It's a bit of a redundant way to solve it, but it was the easiest. I didn't want to invest too much time to make it more smart etc. since we will need to touch this elements rarely, so I thought it's ok in this case.